doc/page01_info.md

```diff
@@ -32,17 +32,19 @@ The following instructions are tentatively supported (ie code is incomplete and
 # Retrieving the source
 
 ## Github
+
 **EVE** is available on GitHub and can be retrieved via the following command:
 <br/>
 
 @verbatim
-$ git clone  https://github.com/jfalcou/eve.git
+$ git clone  <https://github.com/jfalcou/eve.git>
```

Comment:
why the < > ? Is it stylistic ?

Reply:
Oops. Definitely an accident. Too many keyboard shortcuts to accidentally hit. Will remove.